app/assets/javascripts/bastion/layouts/nutupane.html

```diff
@@ -31,6 +31,12 @@ <h2 class="col-sm-2">
                typeahead-empty
                typeahead-template-url="components/views/autocomplete-scoped-search.html"/>
         <span class="input-group-btn">
+          <button class="btn btn-default"
+                  type="button"
+                  ng-click='table.search("") && (table.searchCompleted = false)' 
+                  ng-show="table.searchCompleted && table.searchTerm">
+            <i class="fa fa-undo"></i>
```

Comment:
I personally think that an 'x' would be better here as the undo implies reload or refresh.  When I saw this I thought that clicking it would re-run the search.